src/Kiota.Builder/Writers/Php/CodeMethodWriter.cs

```diff
@@ -201,24 +206,12 @@ private void WriteMethodsAndParameters(CodeMethod codeMethod, LanguageWriter wri
                 writer.IncreaseIndent();
                 return;
             }
-            var isVoidable = "void".Equals(isConstructor ? null : conventions.GetTypeString(codeMethod.ReturnType, codeMethod),
-                StringComparison.OrdinalIgnoreCase);
+            var isVoidable = "void".Equals(isConstructor ? null : conventions.GetTypeString(codeMethod.ReturnType, codeMethod), StringComparison.OrdinalIgnoreCase);
             var optionalCharacterReturn = isVoidable ? string.Empty : orNullReturn[0];
-            var returnValue = isConstructor
-                ? string.Empty
-                : $": {optionalCharacterReturn}{conventions.GetTypeString(codeMethod.ReturnType, codeMethod)}";
-            var pathParametersParam = codeMethod.Parameters.OfKind(CodeParameterKind.PathParameters);
-            var requestAdapterParam = codeMethod.Parameters.OfKind(CodeParameterKind.RequestAdapter);
+            var returnValue = isConstructor ? string.Empty : $": {optionalCharacterReturn}{conventions.GetTypeString(codeMethod.ReturnType, codeMethod)}";
             if (isConstructor && codeMethod?.Parent is CodeClass @class && @class.IsOfKind(CodeClassKind.RequestBuilder))
             {
-                var pathParameters = codeMethod.Parameters
-                    .Where(parameter => parameter.IsOfKind(CodeParameterKind.Path))
-                    .Select(parameter => $"{conventions.GetParameterSignature(parameter, codeMethod)}");
-                var pathParamsString = string.Empty;
-                var parameters = pathParameters.ToList();
-                if (parameters.Any()) pathParamsString = $", {string.Join(", ", parameters)}";
-                var pathParametersString = pathParametersParam != null ? $"{conventions.GetParameterSignature(pathParametersParam, codeMethod)}, " : string.Empty;
-                writer.WriteLine($"{conventions.GetAccessModifier(codeMethod.Access)}{(codeMethod.IsStatic ? " static" : string.Empty)} function {methodName}({pathParametersString}{conventions.GetParameterSignature(requestAdapterParam, codeMethod)}{pathParamsString}) {{");
```

Comment:
Deprecates manual ordering of parameters logic to rely on default `BaseCodeParameterOrderComparer`